gc/base/TLHAllocationSupport.cpp

```diff
@@ -419,4 +439,33 @@ MM_TLHAllocationSupport::objectAllocationNotify(MM_EnvironmentBase *env, void *h
 }
 #endif /* OMR_GC_OBJECT_ALLOCATION_NOTIFY */
 
+/**
+ * Restore the TLH if part of it was reserved by the collector during TLH refresh,
+ * and then notify the collector of it.
+ *
+ * TLH must be restored before it's flushed/cleared.
+ */
+void
+MM_TLHAllocationSupport::restoreTLH(MM_EnvironmentBase *env)
```

Comment:
In a few of those methods I'd bring in word Top (or Tail). Perhaps even ...ForGC (as opposed for mutator what TLHs are normally for).


restoreTLHTop
cacheTopRestored